administration/src/index.tsx

```diff
@@ -1,14 +1,13 @@
+import './polyfills'
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+    React.createElement(App), // Don't use <App />, otherwise Babel will destroy IE11 functionality
```

Comment:
Babel will fuck you up!

Reply:
![](https://media1.giphy.com/media/l1J3G5lf06vi58EIE/giphy.gif?cid=ecf05e47duky1hlfuqlpnbttqliz1j506db2h0465dnzpq4p&rid=giphy.gif)